language/move-binary-format/src/deserializer.rs

```diff
@@ -1386,6 +1386,7 @@ fn load_function_def(cursor: &mut VersionedCursor) -> BinaryLoaderResult<Functio
         function,
         visibility,
         is_entry,
+        is_test: false,
```

Comment:
Drive-by comment. Wondering why do we special case this? Couldn't there be a function which asks something like `has_attribute(..., &TEST_ATTRIBUTE)`? This is how we are doing it in the move model.

Reply:
It has been preliminarily deemed to much of a change as the same info can should be discoverable at the adapter during compilation. Keeping it open as a draft until I can confirm that this is the case, but it will likely never leave the draft form.